src/main/scala/cromwell/engine/workflow/WorkflowActor.scala

```diff
@@ -486,6 +490,8 @@ case class WorkflowActor(workflow: WorkflowDescriptor, backend: Backend)
   onTransition {
     case WorkflowSubmitted -> WorkflowRunning =>
       stateData.startMode.get.start(this)
+      publishWorkflowEvent()
+    case _ -> _ => publishWorkflowEvent()
```

Comment:
This isn't quite what I meant in the earlier comment.  The Akka FSM allows for multiple `onTransition` handlers, all of which will be called on a transition.  I was thinking more like:

```scala
onTransition {
  case WorkflowSubmitted -> WorkflowRunning => stateData.startMode.get.start(this)
}

onTransition {
}